grunt_config.js

```diff
@@ -2,6 +2,8 @@ module.exports = {
   // server process.env.XXXX
   memegenUsername: process.env.memegenUsername,
   memegenPassword: process.env.memegenPassword,
+  dbUsername: process.env.dbUsername,
+  dbPassword: process.env.dbPassword,
```

Comment:
Think we should add a comment around these (and the memegen creds above) to indicate that they aren't actually being used by Grunt and that this is more for documentation